go/teams/list.go

```diff
@@ -45,6 +45,113 @@ func getTeamsListFromServer(ctx context.Context, g *libkb.GlobalContext, uid key
 	return list.Teams, nil
 }
 
+func memberNeedAdmin(member keybase1.MemberInfo, meUID keybase1.UID) bool {
+	return member.UserID == meUID &&
+		(member.Role.IsAdminOrAbove() || (member.Implicit != nil && member.Implicit.Role.IsAdminOrAbove()))
+}
+
+// verifyMemberRoleInTeam checks if role give in MemberInfo matches
+// what team chain says. Nothing is checked when MemberInfo's role is
+// NONE, in this context it means that user has implied membership in
+// the team and no role given in sigchain.
+func verifyMemberRoleInTeam(ctx context.Context, member keybase1.MemberInfo, team *Team) error {
```

Comment:
Since this doesn't check a lot of other parts of `MemberInfo` (team id, team name), could you make it take just what it uses instead? `uid`, `role`, and `team`